src/file.cpp

```diff
@@ -57,10 +67,10 @@ namespace Sass {
     // always with trailing slash
     std::string get_cwd()
     {
-      const size_t wd_len = 4096;
       #ifdef _WASM
-        std::string cwd = "/";
+        std::string cwd = get_cwd_from_env();
```

Comment:
Could you elaborate why `getcwd` does not work for wasm here?

Reply:
wasi's libc does not implement getcwd yet